flow/arguments.go

```diff
@@ -0,0 +1,37 @@
+package cli
```

Comment:
This new file needs the license header, e.g. https://github.com/onflow/flow-cli/blob/master/flow/cadence/vscode/vscode.go#L1-L17.

The `check-headers.sh` script is broken, could you please copy the fixed version from https://github.com/onflow/cadence/blob/master/check-headers.sh and add it as part of this PR? It should detect the missing license header